plugins/msg-broker/mcollective/lib/openshift/mcollective_application_container_proxy.rb

```diff
@@ -3315,7 +3315,7 @@ def self.select_best_fit_node_impl(server_infos)
       # Sort by district available capacity and take the best half
       server_infos = server_infos.sort_by { |server_info| (server_info.district_available_capacity ? server_info.district_available_capacity : 1) }
```

Comment:
The first sort is by node_consumed_capacity, where "less (consumed) is better", so we take the first half.
The second sort is by district_available_capacity, where "more (available) is better", so we take the last half.